pair-programming/week-7/react-express-app/starter-code/React-express-starter-code/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

Comment:
Is this intentional? Generally you want to check this file in to strive for a consistent environment.

Reply:
woops, that's on me I was tinkering with the dependabot PRs and wanted to temporarily remove the package-lock files from the repo, this has been corrected on the main branch and should be resolved in the PR once the branch is updated with main

Reply:
Thanks Andy, thanks Daaimah--just wanted to let you all know that this was seen :) I am now updating the pr.

Reply:
```suggestion
```